01week/datatypes.js

```diff
@@ -0,0 +1,115 @@
+'use strict'
+
+// logs current date
+
+let displayDate = function() {
+	let date = new Date();
+	console.log(date);
+	console.log(typeof(date));
+}
+
+displayDate();
+
+
+// convert number to string
+
+let convertNumber = function() {
+	let newNumber = 5;
+	toString(newNumber);
+	console.log(newNumber);
+	console.log(typeof(newNumber));
+}
+
+convertNumber();
+
+
+// Convert string to number
+
+let convertString = function(x) {
+	parseInt(x);
+	console.log(x);
+	console.log(typeof(x));
```

Comment:
you did the conversion from String to number correctly, but this still prints out "string" when you run the ```console.log(typeof(x))```. What should change so it prints out "number"